data/Trikcode.json

```diff
@@ -0,0 +1,49 @@
+{
+  "name": "Nobert Patrick",
+  "type": "personal",
+  "displayStatsPublic": true,
+  "bio": "Full Stack Developer | **Open Source Contributor**",
+  "avatar": "https://github.com/Trikcode.png",
+  "tags": [
+			"Python",
+			".NET/C#",
+			"Javascript",			
+			"Typescript",
+			"Open Source"
+  ],
+  "socials": [
+    { "icon": "FaTwitter", "url": "https://twitter.com/trikcode" },
+    { "icon": "FaGithub", "url": "https://github.com/Trikcode" }
+  ],
+  "links": [
+    {
+      "name": "Twitter: Follow me",
+      "url": "https://twitter.com/trikcode",
+      "icon": "FaTwitter",
+      "color": "#00ACEE"
+    },
+    {
+      "name": "YouTube: Check out my YouTube Channel",
+      "url": "https://www.youtube.com/channel/UCryU2yhdMXsD5JHed3D0VkA",
+      "icon": "FaYoutube",
+      "color": "#FF0000"
+    },
+    {
+      "name": "GitHub: Let's collaborate",
+      "url": "https://github.com/Trikcode",
+      "icon": "FaGithub"
+    },
+				{
+      "name": "LeetCode: Check Me out",
+      "url": "https://leetcode.com/NobertPatrick/",
+						"icon": "FaLeetcode"
```

Comment:
Same goes for the indentation of `icon` here

Reply:
We have a GitHub Action that will fix the formatting after merge as long as the json is valid - we have a GitHub Action that also check the json is valid in the pull request before merging